cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -249,6 +257,10 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
                     ortResult.getPackageLicenseChoices(pkg.id)
                 ).any { it in includedLicenseCategories }
             }
+
+            if (isModified) {
+                log.info { "Filtered $originalCount to ${packages.size} package(s) by license classification." }
```

Comment:
This can be misleading if some of those packages have been filtered by the id filter already. Maybe a sequential log like this would be better:
```
Found 100 packages.
Removing 10 packages which do not match the id filter.
Removing 20 packages which do not match the license classification filter.
Downloading remaining 70 packages.
```